### PR TITLE
SS13: Pacifist route

### PR DESCRIPTION
**Pull request by ghost**
From #31926:

> @Kurfursten commented 6 hours ago
> For the record, "Voluntary Pax Implant" could be a cool starting character option like disabilities on the list.

And so it was done.
[content][tested]

:cl:
 * rscadd: Players can now start the round with a pacifist mindset.
 * rscadd: Players can now catch the pacifist disability from mutations, which prevents fighting.

### CODE REVIEW

File: __DEFINES/setup.dm [hunk 1]
```diff
@@ -1700,7 +1702,7 @@ var/proccalls = 1
 #define ESPORTS_CULTISTS "Team Geometer"
 #define ESPORTS_SECURITY "Team Security"
 
-#define DNA_SE_LENGTH 58
```
Comment (boy2mantwicethefam) [inline __DEFINES/setup.dm:1703]: Shouldn't this be 59?

**Comment by Optimism333** [comment]
Personally I prefer the idea of a roundstart pax implant disability option so it's a bit harder to remove than a stray rye pill, but I'll take this if this is what we get.

File: __DEFINES/setup.dm (hunk 1 above)
Comment (ghost) [reply __DEFINES/setup.dm:1703]: @robintime forgot to make it that in her anemia PR so I added 2 to cover it instead. (it was causing a warning on server bootups)

**Comment by Eneocho** [comment]
Dangerously based and peacepilled. Can't wait for the inevitable round where the tator forces everyone to pacifism.

**Comment by Kurfursten** [comment]
I also think it would be more fun as an implant but this is also good

**Comment by kilozombie** [comment]
Implant is nice for practicality-- starting with deaf or mute or fr*nch means a trip to Medbay for some rye, and it'd be a shame to lose a character gimmick for the round because of that.

**Comment by Theophrastus1** [comment]
Pacifist Implants for greytide shitters WHEN

**Comment by Optimism333** [comment]
> Implant is nice for practicality-- starting with deaf or mute or fr*nch means a trip to Medbay for some rye, and it'd be a shame to lose a character gimmick for the round because of that.

Yeah, I completely forgot about this. Given that any round might start with a crappy mutation that you'll want to get rid of, there's even more reason to not have this coded in such a way that a dose of rye would remove it.

Please consider making it a round-start implant, kane-f. It'd be better mechanically.

**Comment by coolguymcman** [comment]
looks good, can this get merged?

**Comment by Kurfursten** [comment]
Do you plan to make this an implant or no?

**Comment by Eneocho** [comment]
> Pacifist Implants for greytide shitters WHEN

Go to the vox trader and buy their pax implants.

**Comment by Armadingus** [comment]
Hey fucker why do you keep sneakycoding shit into the game? WHO ASKED FOR TOURETTES HUH???

**Comment by Eneocho** [comment]
Come on @kane-f, you should know by know sneakycodershit is reverted.

**Pull request merged**